### DISCUSSION TITLE
Dynamically add rows to datagrid and populate each row with some default data

### DISCUSSION THREAD

**Discussion by alateefah** [formio/formio.js#5167] [Q&A]
I have an array of objects and would like to use it to create rows in a datagrid. I am using the instance.addRow() to add the row but I also want to set some values for each row from the object.

```
for (let i = 0; i < expected; i++)  {
      //what I would like to do is to set some values like this
      //row.field = "s"; //not fixed though. I have the data in the object
      //before adding the row
      instance.addRow();
 }
```

**Comment by HCMF** [comment]
I need to do the same thing... Did you ever find an answer?
